cfpb/owning-a-home
# Email signup popup

Adds email signup popup on `/process/prepare`, `/process/compare`, and `/process/explore`

## Additions

- Email signup popup

## Testing

- Check out branch & run grunt
- Navigate to http://localhost:8000/owning-a-home/process/prepare/, http://localhost:8000/owning-a-home/process/explore, or http://localhost:8000/owning-a-home/process/compare/ and scroll down the page. Popup should appear after you've scrolled about halfway down.
- Reload the page or navigate to another page. Popup should not appear after scrolling down page. 

## Review

- @sebworks

## Screenshots


## Checklist
